.github/workflows/build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v2
         with:
-          node-version: 16.15
+          node-version: 20.15
```

Comment:
Necessary because I got a `ReferenceError: [plugin vite-plugin-svelte] structuredClone is not defined` error on the build workflow while building the Svelte 5 playground.